### PR TITLE
Fix reset form field alias

### PR DESCRIPTION
**Pull request by kuzmany**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.2
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 

#### Description:
Noticed issue  reset custom alias of field was reset every time while we edit it

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create new form
3. Create new field **Custom** with Field HTML name **custom1**
4. Save it
5. Open it and see Field HTML name is still **custom1**
6. Save again and see Field HTML name is now **custom**
7. We expect still **custom1** alias of field

image

### CODE REVIEW

**Review by escopecz — CHANGES_REQUESTED** [review]
Plus needs a test

File: app/bundles/FormBundle/Controller/FieldController.php
Comment (escopecz) [inline app/bundles/FormBundle/Controller/FieldController.php]: Hmm, my suggestion was not saved for some reason. Trying once more
```suggestion
                                $formField['alias'] ?? $formField['label'],
```

**Review by escopecz — CHANGES_REQUESTED** [review]

**Review by mohit-rocks — APPROVED** [review]
I did a quick test on local and it seems working fine.
Attaching small video recording for the same:

https://www.loom.com/share/4ec35c10f2f748ff90ef6bbf23f8f868

**Comment by RCheesley** [comment]
@escopecz is this one good to go now? I know you had a few suggestions earlier in the process so just checking that this is good from your perspective before we merge

**Review by escopecz — DISMISSED** [review]
I see no issue. Thanks Zdeno! 👍

**Review by escopecz — COMMENTED** [review]
Although, double-checking the test. It does not test the change in the FieldController but the `generateAlias` method in the FormModel. It doesn't make much sense.

**Comment by escopecz** [comment]
I don't want to block this PR. If the release leader thinks it's OK to test different part of the code than which was changed to get the green light from Codecov check then feel free to merge.

**Comment by RCheesley** [comment]
I will allow this one to pass having tested and confirmed that the PR does fix the reported issue, however in the future please  @kuzmany take on board the [comments from John](https://github.com/mautic/mautic/pull/9443#pullrequestreview-588433745)

**Pull request merged**
